pkg/ingester/ingester.go

```diff
@@ -654,30 +730,21 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	ingestionRateTicker := time.NewTicker(instanceIngestionRateTickInterval)
 	defer ingestionRateTicker.Stop()
 
```

Comment:
> I think it's correct to track such stats. I don't see why during a catch up we don't want to enforce the MaxIngestionRate. The instance limit should be set to protect the ingester, and we should protect the ingester regardless it's in starting or running state.

The protection is against many tenants sending too much data at once. With Kafka in the middle, Kafka can become overloaded, but I don't see how ingester could "overload" itself during replay, since it's in the same process, and also replay is single-threaded. I think this per-instance rate-limit makes little sense with ingest-storage, but we can simply disable it via config, so it's not a big issue.